### ISSUE TITLE
Masking policy with multiline statement updated every time

### ISSUE THREAD

**Issue by kanomaxb** [Snowflake-Labs/terraform-provider-snowflake#1097]
**Provider Version**

0.36

**Describe the bug**

When masking policy is created using multiline string, like this:
```
  masking_expression = <<EOF
        case 
            when current_role() in ('ROLE_A') then 
                val 
            when is_role_in_session( 'ROLE_B' ) then 
                'ABC123'
            else
                '******'
        end
    EOF
```
then it is updated each time. 
The plan claim that it needs to replace `case` with the whole expression

**Expected behavior**

Plan should be empty when no change in masking expression is made.

![image](https://user-images.githubusercontent.com/87711035/177332975-47d9314c-6eed-4de8-83cc-66ed71b27250.png)

**Comment by jaloren** [comment]
It looks like when the masking policy is stored in snowflake, it will truncate newlines on the last line of text. If you wrap the multi-line string in a chomp, then terraform shows no difference.

**Comment by sfc-gh-jmichalak** [comment]
Hi, I'm sorry for the late response. We've released [v0.96.0](https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases/tag/v0.96.0), which includes adjusting and fixing row access policies. Please upgrade with the [migration guide](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md#v0950--v0960).

**Comment by sfc-gh-jmichalak** [comment]
Closing the issue due to inactivity.
